packages/admin-ui/src/skeleton/skeleton.style.ts

```diff
@@ -0,0 +1,44 @@
+import { keyframes, style, styleVariants } from '@vtex/admin-ui-core'
+
+export const variants = styleVariants({
+  shape: {
+    rect: {
+      borderRadius: 'default',
+    },
+    circle: {
+      borderRadius: 'circle',
+    },
+  },
+})
+
+const waveKeyframe = keyframes({
+  '0%': {
+    transform: 'translateX(-100%)',
+  },
+  '50%': {
+    transform: 'translateX(100%)',
+  },
+  '100%': {
+    transform: 'translateX(100%)',
+  },
+})
+
+export const baseline = style({
+  display: 'block',
+  position: 'relative',
+  width: 'full',
```

Comment:
Do we should use as `$full`?

Reply:
I'll change to `100%` instead since we don't have plans to define size tokens for now